integration_tests/suite_test.go

```diff
@@ -60,6 +62,7 @@ type IntegrationTest struct {
 }
 
 var ITest IntegrationTest
+var vaultTestCluster *vault.TestCluster
```

Comment:
I'd like it better if this was part of the `ITest` global, so that we have just one place to find all the "support infrastructure" for the tests

Reply:
done